### PR TITLE
Add updates-flatpak

### PR DESCRIPTION
**Pull request by j2ghz**
I couldn't find a better way.
If there are no updates, `flatpak update` output has 2 lines. 
If there are some, it's 5 + number of updates.

I also made build.sh executable

### CODE REVIEW

**Comment by x70b1** [comment]
> I also made build.sh executable

Whats the reason for this? 
Travis works without it.

The rest looks nice! I have to test it first as I don't use flatpak. Give me some time for it.

**Comment by j2ghz** [comment]
> > I also made build.sh executable
>
> Whats the reason for this?

I cloned the repository locally, made my changes, and when I wanted to run `build.sh`, I had to run `chmod +x build.sh` before. It's easier for contributors if it's executable right from the start.

**Comment by x70b1** [comment]
Just use `sh build.sh`?

**Comment by j2ghz** [comment]
The most common way to execute a script is to do `./build.sh`, and that doesn't work if the file is not executable.

**Comment by x70b1** [comment]
I don't want to discuss here what the common way is for the execution.
At least it should be in its own pull request.

File: polybar-scripts/updates-flatpak/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Script: updates-flatpak
+
+A script that shows if there are updates for FlatPak.
+
+
+## Dependencies
+
+* `flatpak` to run `flatpak update`
+
```
Comment (x70b1) [inline polybar-scripts/updates-flatpak/README.md]: I guess we can delete this section. This is very obvious.

File: polybar-scripts/updates-flatpak/updates-flatpak.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+if ! updates=$(echo "n" | flatpak update 2> /dev/null | wc -l ); then
```
Comment (x70b1) [inline polybar-scripts/updates-flatpak/updates-flatpak.sh]: We should think about using `flatpak update --noninteractive`? This might be cleaner.

Comment (j2ghz) [reply polybar-scripts/updates-flatpak/updates-flatpak.sh]: I tried using that and the problem is it just runs the update. `flatpak update --help` doesn't list any way to only print updates.

Comment (x70b1) [reply polybar-scripts/updates-flatpak/updates-flatpak.sh]: Mh k, that's annoying. I had tested it, but I have not tried it with updates.

**Comment by x70b1** [comment]
@j2ghz Thanks for this! :+1:

**Pull request merged**